gap/Utils.gi

```diff
@@ -66,6 +66,37 @@ function(M, func)
     return result;
 end);
 
+# Take an m x n-matrix M and change its shape to a rows x cols-matrix, where
+# entries are copied column by column. 
+BindGlobal("ReshapeMatrix",
```

Comment:
This is not used anymore... THough perhaps it'll be handy elsewhere later on?

Reply:
Perhaps - I guess it doesn't do much harm so ...